### PR TITLE
Fixing lineGraph ordering issue

### PR DESCRIPTION
**Pull request by krashish8**
lineGraph on #68.

Changes proposed in this pull request:
- Adds pgTAP tests for lineGraph directory that check the output after rows ordering.

@pgRouting/admins

### CODE REVIEW

**Comment by krashish8** [comment]
For this function, if I sort the rows, then the `lineGraphFull-reverse_cost_equivalence.sql` pgTAP test fails.

File: pgtap/lineGraph/lineGraph/lineGraph-rows-check.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+PREPARE randomOrderUndirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()',
+    directed => false
+);
+
+SELECT set_eq('expectedOutputDirected', 'descendingOrderDirected', '1: Should return same set of rows');
```
Comment (cvvergara) [inline pgtap/lineGraph/lineGraph/lineGraph-rows-check.sql:54]: ./../pgtap/lineGraph/lineGraph/lineGraph-rows-check.sql                                             (Wstat: 0 Tests: 20 Failed: 10)
  Failed tests:  1-5, 11-15
Mark the failed tests with `todo`

File: pgtap/lineGraph/lineGraphFull/lineGraphFull-rows-check.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+);
+
+PREPARE randomOrder AS
+SELECT * FROM pgr_lineGraphFull(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()'
+);
+
+SELECT set_eq('expectedOutput', 'descendingOrder', '1: Should return same set of rows');
```
Comment (cvvergara) [inline pgtap/lineGraph/lineGraphFull/lineGraphFull-rows-check.sql:30]: ../../pgtap/lineGraph/lineGraphFull/lineGraphFull-rows-check.sql                                     (Wstat: 0 Tests: 10 Failed: 10)
  Failed tests:  1-10
.Martk the failed tests with todo

**Comment by cvvergara** [comment]
Both functions are experimental so dont bother changing code

**Pull request merged**